### DISCUSSION TITLE
How stretch my brand logo on the Top-Left corner

### DISCUSSION THREAD

**Discussion by noblemfd** [ColorlibHQ/AdminLTE#3432] [Q&A] [answered]
From the AdminLTE-v3, I want to adjust the brand-logo to stretch and file the top-left corner and also remove the span:

This is how it appears originally:

![image](https://user-images.githubusercontent.com/34077129/108485146-e2c5a380-729c-11eb-97da-21c18a7266af.png)

I have a Brand Logo with wide width. I want to remove the AdminLTE3 and make my brand logo to stretch across.

    `<aside class="main-sidebar sidebar-dark-primary elevation-4">
    
    <a href="index3.html" class="brand-link">
      <img src="dist/img/AdminLTELogo.png" alt="AdminLTE Logo" class="brand-image img-circle elevation-3"
           style="opacity: .8">
      <span class="brand-text font-weight-light">AdminLTE 3</span>
    </a>`

From the above, when I removed

`<span class="brand-text font-weight-light">AdminLTE 3</span>`

o achieve this, everything shrinks.

How do I achieve my goal?

Thanks

**Comment by REJack** [comment, marked as answer]
You want to use a wide image?

**Reply by noblemfd** [reply]
@REJack - I just want the image to stretch across that width (maybe 200px) and also remove <span class="brand-text font-weight-light">AdminLTE 3</span>. Also each time I remove the span, it distort the whole thing.

Something like this:

![image](https://user-images.githubusercontent.com/34077129/108488920-3e922b80-72a1-11eb-93a1-f8320b9b101e.png)

**Reply by REJack** [reply]
Look at the [docs](https://adminlte.io/docs/3.0/components/main-sidebar.html) at _Alternate Logo_.

**Reply by REJack** [reply]
P.S. The docs uses the _Alternate Logo_.

**Reply by noblemfd** [reply]
@REJack - I did as you advised. It works. But when I got to some pages, that the logo did not come out but displays AdminLTE Docs Logo Large as in alt="AdminLTE Docs Logo Large"

I mean as I navigate some come out like this even with the same layout:
![image](https://user-images.githubusercontent.com/34077129/108505965-6b9e0880-72b8-11eb-9d29-c9a0c516d9ce.png)

While others are fine

**Reply by REJack** [reply]
If the alt text is displayed, then the imgae file isn't found.

**Reply by noblemfd** [reply]
@REJack - The same thing I did all-round:

    @include('layouts.partials.brand-logo')

for

    <a href="{{ route('home') }}" class="brand-link logo-switch navbar-dark bg-gray-light">
      <img src="adminlte3/dist/img/logo.png" alt="AdminLTE Docs Logo Small" class="brand-image-xl logo-xs">
      <img src="adminlte3/dist/img/logo.png" alt="AdminLTE Docs Logo Large" class="brand-image-xs logo-xl" style="left: 12px">
    </a>

Some display while others did not

**Reply by REJack** [reply]
From the code it's correct, but the path's I can't check 😄

**Reply by noblemfd** [reply]
@REJack - See the path:

![image](https://user-images.githubusercontent.com/34077129/108532034-acf1e080-72d7-11eb-9ae3-93af0df2bb38.png)
